### ISSUE TITLE
data seeding/fixtures

### ISSUE THREAD

**Issue by unenglishable** [epochtalk/epochtalk-api-tests#1]
to help with complex tests, consider adding a seeding/fixture scheme.

**Comment by wangbus** [comment]
is this still relevant?

**Comment by unenglishable** [comment]
@wangbus yes, it is.  seeding for each test would be much easier if done by db call, but that will require tighter integration with the main epochtalk project.

in particular, it would be great to create and destroy data in `before` and `after` clauses using each module's `db.create`, `db.delete` etc...  rather than using api-only interactions

this would reduce the time it takes to write a test, as well as the time it takes for the tests to run.
